apps/remix-ide/src/app/ui/landing-page/landing-page.js

```diff
@@ -466,9 +472,38 @@ export class LandingPage extends ViewPlugin {
           <div class="d-flex justify-content-between">
             <div class="d-flex flex-column">
               <div class="border-bottom d-flex justify-content-between clearfix py-3 mb-4">
-                <div class="mx-4 w-100">
+                <div class="mx-4 w-100 d-flex">
                   ${img}
                   <audio id="remiAudio" muted=false src="assets/audio/remiGuitar-single-power-chord-A-minor.wav"></audio>
+                  <div class="w-80 pl-5 ml-5">
```

Comment:
Here are 2 updates:
```
<div class="w-80 pl-5 ml-5"><h4 class="mb-1">Quicklinks</h4><a href="https://medium.com/remix-ide/migrating-files-to-workspaces-8e34737c751c?source=friends_link&amp;sk=b75cfd9093aa23c78be13cce49e4a5e8">Guide for migrating</a> your files to a File Explorer workspace
<h6 class="mt-1 mb-1">Migration tools:</h6><li><u class="text_2A2Ew px-1">Download all Files</u></li><li><u class="text_2A2Ew px-1">Restore Backup Zip</u>- a plugin to current workspace</li> <h6 class="mt-1 mb-0">Help</h6><li><a target="__blank" href="https://gitter.im/ethereum/remix" class="text_2A2Ew mx-1">Gitter channel</a></li><li><a target="__blank" href="https://github.com/ethereum/remix-project/issues" class="text_2A2Ew mx-1">Report on Github</a></li></div>
```
or without bullet points
```
<div class="w-80 pl-5 ml-5"><h4 class="mb-1">Quicklinks</h4><a href="https://medium.com/remix-ide/migrating-files-to-workspaces-8e34737c751c?source=friends_link&amp;sk=b75cfd9093aa23c78be13cce49e4a5e8">Guide for migrating</a> your files to a File Explorer workspace
<h6 class="mt-1 mb-1">Migration tools:</h6><p class="mb-0"><u class="text_2A2Ew px-1">Download all Files</u></p><p class="mb-2"><u class="text_2A2Ew px-1">Restore Backup Zip</u>- a plugin to current workspace</p> <h6 class="mt-1 mb-0">Help</h6><p class="mb-0"><a target="__blank" href="https://gitter.im/ethereum/remix" class="text_2A2Ew mx-1">Gitter channel</a></p><p class="mb-0"><a target="__blank" href="https://github.com/ethereum/remix-project/issues" class="text_2A2Ew mx-1">Report on Github</a></p></div>
```

Reply:
I've put ballots to show that it is 2 step action not just some links